sdk/documenttranslation/Azure.AI.DocumentTranslation/src/TranslationStatusResult.cs

```diff
@@ -7,50 +7,73 @@
 
 namespace Azure.AI.DocumentTranslation
 {
+    /// <summary>
+    /// Status information about the translation operation.
+    /// </summary>
     [CodeGenModel("BatchStatusDetail")]
     public partial class TranslationStatusResult
     {
-        /// <summary> Id of the operation. </summary>
+        /// <summary>
+        /// Id of the translation operation.
+        /// </summary>
         [CodeGenMember("Id")]
         public string TranslationId { get; }
 
-        /// <summary> Operation created date time. </summary>
+        /// <summary>
+        /// The date time when the translation operation was created.
+        /// </summary>
         [CodeGenMember("CreatedDateTimeUtc")]
         public DateTimeOffset CreatedOn { get; }
 
-        /// <summary> Date time in which the operation's status has been updated. </summary>
+        /// <summary>
+        /// The date time when the translation operation's status was last updated.
+        /// </summary>
         [CodeGenMember("LastActionDateTimeUtc")]
         public DateTimeOffset LastModified { get; }
 
-        /// <summary> The Status Summary of the operation. </summary>
-        [CodeGenMember("Summary")]
-        internal StatusSummary Summary { get; set; }
-
-        /// <summary> Total count. </summary>
+        /// <summary>
+        /// Total number of documents.
```

Comment:
I think this should be worded more like "total number of translations". E.g. if 2 documents will be translated to 3 different languages, the total returned will be 6.

Reply:
Great point.

Reply:
I am thinking of adding `documents` to it otherwise the name of the property is weird...

Reply:
let me know if this is better

Reply:
looks good!